pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -544,10 +544,10 @@ func (cm *containerManagerImpl) setupNode(activePods ActivePodsFunc) error {
 	return nil
 }
 
-func getContainerNameForProcess(name, pidFile string) (string, error) {
-	pids, err := getPidsForProcess(name, pidFile)
+func getContainerNameForProcess(names []string, pidFile string) (string, error) {
```

Comment:
What's the point to change type of the name parameter to slice if this function is called with only one process name?
```
cont, err := getContainerNameForProcess([]string{dockerProcessName}, dockerPidFile)
...
cont, err := getContainerNameForProcess([]string{dockerProcessName}, dockerPidFile)
```

Reply:
@bartosh I want it would work for both 'containerd' and 'docker-containerd' in this case.

Reply:
@pacoxu Can you show where do you call getContainerNameForProcess with 'containerd' and 'docker-containerd'?

Reply:
per comments, changed process name only.